contracts/RemovalUtils.sol

```diff
@@ -44,9 +44,8 @@ uint256 constant _ASCII_CAP_LETTER_MAX_VAL = 90;
  *
  * Version 0:
  * [1byte][1byte][--2 bytes--][--2 bytes--][--2 bytes--][----------- 20 bytes------------- ][------4 bytes------]
- * tokIdV--meth&v---vintage------country------subdivision------------ supplier address --------------subidentifier--
+ * tokIdV--meth&v---vintage------country------subdivision------------ supplier address --------------projectId--
```

Comment:
:tada: 

Btw, is there logic around how many dashes each of these components of the ID use in the natspec? (e.g., why `--2 bytes--` where there are 4 dashes and `----------- 20 bytes-------------` where there are 24 dashes?

Reply:
ah, was looking at it from a view that was not lining up visually